prow/config.yaml

```diff
@@ -98,14 +98,29 @@ branch-protection:
         istio.io:
           branches: *blocked_branches
         operator:
+          required_pull_request_reviews: &tide_reviews
+            require_code_owner_reviews: true
+          restrictions: &tide_restrictions
+            teams:
+              - repo-admins
           branches: *blocked_branches
         cni:
           branches: *blocked_branches
         installer:
           branches: *blocked_branches
         pkg:
+          required_pull_request_reviews: &tide_reviews
+            require_code_owner_reviews: true
+          restrictions: &tide_restrictions
+            teams:
+              - repo-admins
           branches: *blocked_branches
         common-files:
+          required_pull_request_reviews: &tide_reviews
```

Comment:
This anchor has already been defined on L106/L112, so this should just be `required_pull_request_reviews: *tide_reviews`